packages/kit/src/core/prerender/prerender.js

```diff
@@ -136,7 +136,12 @@ export async function prerender({ config, entries, files, log }) {
 			}
 		});
 
-		const text = await response.text();
+		let text = null;
+		if (response?.headers.get('Content-Type') && (response.headers.get('Content-Type')).startsWith('image/')) {
```

Comment:
you should use the existing `is_text` method: https://github.com/sveltejs/kit/blob/d430a33643b33f6caa0b0eb69ed558175dbcb343/packages/kit/src/runtime/server/endpoint.js#L32

Reply:
What would happen if this just used the `arrayBuffer` version all of the time? Is there a reason to pass a string to `save()` sometimes?

Reply:
yeah, that's probably an even better idea assuming there's nothing that it prevent it from working

Reply:
That all sounds like a good idea. Should I leave it with you and close the PR ? it seems like further down, lint doesn't like body to be not a string either.